database/src/main/postgres/runs/V1.8.3__get_partitioning_checkpoint.sql

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- Function: runs.get_partitioning_checkpoints(JSONB, INT, TEXT)
+CREATE OR REPLACE FUNCTION runs.get_partitioning_checkpoints(
```

Comment:
name of this file does not correspond to the function name - rename this file to plural of the word checkpoint

Reply:
Ha missed that one, good catch.